src/stubgen.py

```diff
@@ -178,7 +178,8 @@ def __init__(
         max_expr_length: int = 50,
         patterns: List[ReplacePattern] = [],
         quiet: bool = True,
-        output_file: Optional[Path] = None
+        output_file: Optional[Path] = None,
+        member_sort: Optional[Union[Callable[[Tuple[str, Any]], Any], Literal['as-defined']] = None,
```

Comment:
```suggestion
        member_sort: Optional[Union[Callable[[Tuple[str, Any]], Any], Literal['as-defined']]] = None,
```
My bad, I dropped a closing bracket in my suggestion